e3sm_to_cmip/__main__.py

```diff
@@ -639,6 +643,19 @@ def _setup_dirs_with_paths(self):
 
         tempfile.tempdir = temp_path
 
+    def get_handler_info_msg(self, handler):
+        msg = {
+            "CMIP6 Name": handler["name"],
+            "CMIP6 Table": handler["table"],
+            "CMIP6 Units": handler["units"],
+            "E3SM Variables": ", ".join(handler["raw_variables"]),
+        }
+        if handler.get("unit_conversion"):
+            msg["Unit conversion"] = handler["unit_conversion"]
+        if handler.get("levels"):
+            msg["Levels"] = handler["levels"]
+        return msg
+        
```

Comment:
```suggestion
```

This looks like a duplicate of `util.get_handler_info_msg` and is not being used in this module. 

It can be deleted.